cmd/fission-cli/app/app.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/fission/fission/pkg/fission-cli/cliwrapper/driver/cobra/helptemplate"
 	"github.com/fission/fission/pkg/fission-cli/cmd"
 	"github.com/fission/fission/pkg/fission-cli/cmd/canaryconfig"
+	"github.com/fission/fission/pkg/fission-cli/cmd/check"
```

Comment:
Looks like we are missing copyright in this file. Can you please add it ?
Reference: https://github.com/fission/fission/blob/3ac188124e348c2463a9e8b46bae9ab8967ce255/pkg/utils/gitrepo/gitrepo.go#L1